Fix KKUx ES VM security groups [BB-4818] [WIP]

Cup0fCoffee opened the pull request:
Add new security group which allows port used by Prometheus for monitoring.

Cup0fCoffee committed:
Add new security group for Prometheus monitoring

Add new security group which allows port used by Prometheus for
monitoring.

giovannicimolin commented on modules/openedx/application/security-groups.tf:
```diff
@@ -0,0 +1,62 @@
+
+resource "aws_security_group" "allow_prometheus" {
+  // TODO: change name and description
+  name        = "allow_prometheus"
```
@Cup0fCoffee Change this to use the same naming convention being used on this folder.

Cup0fCoffee committed:
Follow conventions for "allow_prometheus" sg

Keep the naming for "allow_prometheus" security group with the existing
implementation.

giovannicimolin committed:
Nit: minor fix

giovannicimolin commented:
@Cup0fCoffee I've applied one extra change since I've had some issues while applied, but none other than that.

It worked perfectly after adding the `cidr` lines.

giovannicimolin reviewed (approved):
:+1:

- [x] I tested this:
1. Applied these changes on KKUx production.
- [x] I read through the code
- [ ] I checked for accessibility issues
- [ ] Includes documentation

@Cup0fCoffee Worked perfectly after the small fixes.
Thanks for the work here even without access to the production environment.

Good to merge and tag a release.
